docs/csharp/programming-guide/classes-and-structs/snippets/properties/HidingProperty.cs

```diff
@@ -38,7 +38,7 @@ public static void Test()
         ((Employee)m1).Name = "Mary";
         //</CastToBase>
 
-        System.Console.WriteLine("Name in the derived class is: {0}", m1.Name);
+        System.Console.WriteLine($"Name in the derived class is: {m1.Name}");
         System.Console.WriteLine("Name in the base class is: {0}", ((Employee)m1).Name);
```

Comment:
Change this one too?

```suggestion
        System.Console.WriteLine($"Name in the base class is: {((Employee)m1).Name}");
```